communication/http/get-http-query-uri.yml

```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: get HTTP query URI
```

Comment:
I'm not sure about "query"... How about request or header?

Reply:
tbh, i misunderstood what this `HttpQueryInfo` option fetched. i thought it was the query parameters portion of the URL, but i see now its the whole URL. so i'd propose:

`get HTTP request URL`